hypervisor/src/vm.rs

```diff
@@ -370,4 +390,8 @@ pub trait VmOps: Send + Sync {
     fn pio_read(&self, port: u64, data: &mut [u8]) -> Result<()>;
     #[cfg(target_arch = "x86_64")]
     fn pio_write(&self, port: u64, data: &[u8]) -> Result<()>;
+    #[cfg(feature = "sev_snp")]
```

Comment:
Why do these have to be in `VmOps` - where are they called from that makes that the case?

Reply:
Checkout this PR for reference: https://github.com/cloud-hypervisor/cloud-hypervisor/pull/5857

Reply:
It would be better to include this commit in the POR that uses it then.

Reply:
I don't see why this needs to bounce make from the hypervisor layer into the Vm layer - the only thing that it consumes is the Vm fd. Can't you handle this inside the hypervisor layer itself?

Reply:
Umm then I would need to add vm_fd to MshvVcpu struct as this is handled as part of vcpu run loop.

Reply:
In our implementation vmfd is part of the MshvVcpu, Isn't it?

Reply:
Rob has a fair point. You can simply extend `MshvVcpu` to include the Vm fd along with small modification to `create_vcpu()` to avoid the bounce between the hypervisor crate and vmm crate.

Reply:
Done moved it to use vm_fd instead of using vm_ops.